packages/sakuli-legacy/src/loader/legacy-project-properties.class.ts

```diff
@@ -140,7 +140,7 @@ export class LegacyProjectProperties {
      * Sakuli click delay when clicking mouse buttons in ms
      */
     @NumberProperty('sakuli.clickDelay')
-    sikuliClickDelay: number = 0.2;
+    sikuliClickDelay: number = 200;
```

Comment:
We could get rid of this whole property by providing an alternative property name to the new `mouseActionDelay` property.

E.g.

```
@NumberProperty('sakuli.mouse.action.delay')
@NumberProperty('sakuli.clickDelay')
mouseActionDelay: number = 200;
```

Reply:
I like! :+1: